From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ksasagit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@ksasagit Thank you for this contribution.  I'll have the author review.

@dlepow Would you please review this contribution and let me know if I should merge it?  Thank you.

dlepow commented:
@ksasagit - Thanks for catching this! If you have additional feedback, feel free to contact me. I've had other suggestions for improvement that I'll incorporate soon.
@deneha - Hi Dene, please merge. Thanks!

Commit:
jenny articles

Commit:
updates to docs

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
updates

Commit:
first cut node service TPM

Commit:
Draft quickstart article for creating an Automation account

Commit:
fixed indents at end

Commit:
first cut of node X509 service

Commit:
spelling fix

Commit:
feedback:

Commit:
Merge branch 'release-automation-mvc' of https://github.com/MicrosoftDocs/azure-docs-pr into release-automation-mvc-v2

Commit:
various updates

Commit:
Merge pull request #1 from georgewallace/csand-msft

feedback : george

Commit:
updates to article

Commit:
updates to images

v-shils committed:
Merge pull request #29996 from csand-msft/release-automation-mvc

Draft quickstart article for creating an Automation account

Commit:
Merge branch 'release-automation-mvc' of https://github.com/MicrosoftDocs/azure-docs-pr into release-automation-mvc-v2

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
removed TBDs

Commit:
removed TBDs

Commit:
updates to articles and images

Commit:
updates to link

Commit:
spelling mistake

Commit:
updates to images

Commit:
correcting list

Commit:
correcting list

Commit:
update

Commit:
corrected blocking and nonblocking issues

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

tynevi committed:
Merge pull request #29963 from georgewallace/release-automation-mvc-v2

Automation : MVC updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

BryanLa committed:
modifications for UA MSI

BryanLa committed:
updates

PRMerger16 committed:
Merge pull request #30168 from BryanLa/bryanla-release-msi-tutorial

Bryanla release msi tutorial

BryanLa committed:
updates

PRMerger12 committed:
Merge pull request #30173 from BryanLa/bryanla-release-msi-tutorial

Tutorial updates

BryanLa committed:
updates

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
Merge pull request #30176 from BryanLa/bryanla-release-msi-tutorial

Bryanla release msi tutorial

BryanLa committed:
updates

BryanLa committed:
updates

Commit:
added code

Commit:
removed extra lf in code that caused break

Commit:
updates

Commit:
Merge branch 'release-automation-mvc' of https://github.com/MicrosoftDocs/azure-docs-pr into release-automataion-mvc

Commit:
updates

Commit:
updates

Commit:
testing images

v-nagta committed:
USER STORY 1147269 SaaS App Tutorial: Pluralsight Tutorial Update

Commit:
update sto articles

Commit:
updates to index:

Commit:
Merge pull request #30189 from BryanLa/bryanla-release-msi-tutorial

updates

BryanLa committed:
updates

BryanLa committed:
updates

PRMerger12 committed:
Merge pull request #30274 from BryanLa/bryanla-release-msi-tutorial

Bryanla release msi tutorial

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
initial changes

BryanLa committed:
initial changes

rjagiewich committed:
Merge pull request #30286 from BryanLa/bryanla-release-msi-tutorial

Bryanla release msi tutorial

PRMerger15 committed:
Merge pull request #30289 from BryanLa/bryanla-release-msi-configure

Bryanla release msi configure

BryanLa committed:
updates

BryanLa committed:
updates

Commit:
content/image fixes

Commit:
Merge pull request #30312 from BryanLa/bryanla-release-msi-configure

Bryanla release msi configure

Commit:
updates

BryanLa committed:
updates

BryanLa committed:
updates

PRMerger13 committed:
Merge pull request #30317 from BryanLa/bryanla-release-msi-configure

Bryanla release msi configure

rjagiewich committed:
Update index.yml

BryanLa committed:
updates

Commit:
fixed text/images per feedback

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Lisaco88 committed:
Merge pull request #30327 from BryanLa/bryanla-release-msi-configure

updates

tynevi committed:
Merge pull request #30239 from georgewallace/release-automataion-mvc

automation : updates to mvc docs

v-nagta committed:
updated

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
initial add

BryanLa committed:
Merge branch 'release-msi-user-assigned' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-release-msi-configure

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

v-jasowells committed:
Update toc.yml

Commit:
Improve definition of entities/places

Clarify that entities are well-known (famous). Also add hotels to places.

v-jasowells committed:
Replace example query.

puicchan committed:
Update deploy-jenkins-app-service-plugin.md

Customers are confused and thought that they need to do goth Git and FTP. Updated subheadings to clarify.
“Configure Jenkins to deploy Web Apps through Git and FTP” to “Configure Jenkins to deploy Web Apps by uploading files” 
“Deploy Web Apps through FTP by using Jenkins Pipeline” to “Deploy Web Apps by uploading files using Jenkins pipeline”

puicchan committed:
Update deploy-jenkins-app-service-plugin.md

BryanLa committed:
updates

BryanLa committed:
updates

Ja-Dunn committed:
Merge pull request #30363 from BryanLa/bryanla-release-msi-configure

Bryanla release msi configure

BryanLa committed:
Merge branch 'release-msi-user-assigned' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-release-msi-use

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

johnkemnetz committed:
new eh stuff

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into eventhub-articles

BryanLa committed:
updates

BryanLa committed:
updates

Commit:
fixed Title, Description, H1

Commit:
fixed title, description, H1

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

itechedit committed:
Merge pull request #30420 from BryanLa/bryanla-release-msi-configure

Fix substitution parameters

BryanLa committed:
updates

itechedit committed:
Merge pull request #30367 from BryanLa/bryanla-release-msi-use

initial add

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

Commit:
corrections to some grammar

BryanLa committed:
updates

Ja-Dunn committed:
Merge pull request #30474 from georgewallace/release-automation-mvc

automation : corrections to some grammar

BryanLa committed:
updates

BryanLa committed:
fix inline bash formatting

BryanLa committed:
add example responses

tracsman committed:
Merge branch 'master' into jonorTrouble

tracsman committed:
Merge branch 'jonorTrouble' of https://github.com/tracsman/azure-docs-pr into jonorTrouble

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
fix code block rendering

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

v-alje committed:
Bringing even with master.

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

BryanLa committed:
updates

Commit:
register ACS RP

tracsman committed:
Update with feedback from Osama and Fabrizio

tracsman committed:
Update last edit date

tracsman committed:
Fix Acrolinx issues

rjagiewich committed:
Merge pull request #30490 from BryanLa/bryanla-release-msi-tutorial

fix inline code block formatting

rjagiewich committed:
Merge pull request #30483 from BryanLa/bryanla-release-msi-use

Add sign-in and other updates

BryanLa committed:
initial updates

BryanLa committed:
initial updates

ArLucaID committed:
Added Known issues with private preview

BryanLa committed:
initial updates

BryanLa committed:
toc cleanup

ArLucaID committed:
Fixed acrolinx suggestions.

linda33wj committed:
Update Dynamics data type support

wolfma61 committed:
Update GetStartedJSWebsockets.md

update documentation

rjagiewich committed:
Merge pull request #30507 from BryanLa/bryanla-release-msi-arturo

Initial updates for storage tutorial

BryanLa committed:
fix sign-in text

ArLucaID committed:
Updated with feedback from Jim Wells review

rjagiewich committed:
Merge pull request #30514 from BryanLa/bryanla-release-msi-toc

User-Assigned MSI toc cleanup

rjagiewich committed:
Merge pull request #30513 from arluca/patch-3

Added Known issues with private preview

BryanLa committed:
add pp info to preview banner

v-alje committed:
Bringing even with master.

v-alje committed:
Bringing even with master.

tracsman committed:
Fix typos and formatting

tracsman committed:
Fix Acrolinx issues

tracsman committed:
Fix Acrolinx issues

Commit:
corrected inaccurate headings and link

GitHubber17 committed:
Merge pull request #30541 from georgewallace/release-automation-mvc

automation : corrected inaccurate headings and link

tracsman committed:
Fix astrisks typo

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-quick-service-tpm-node
sync with master

BryanLa committed:
initial updates

BryanLa committed:
updates

Commit:
add selectors, still need to do TOC

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-quick-service-x509-node
sync with master

Commit:
synced with master and added selectors

hning86 committed:
Update tutorial-classifying-iris-part-3.md

johnkemnetz committed:
new eh stuff

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into eventhub-articles

BryanLa committed:
updates

BryanLa committed:
add FAQ item

Commit:
blurred subscription ID in URL

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-quick-service-tpm-node
sync with master

Commit:
added node TPM to TOC

BryanLa committed:
updates

BryanLa committed:
intial updates

GitHubber17 committed:
Merge pull request #30542 from BryanLa/bryanla-release-msi-grant

User-assigned MSI for RBAC grant

BryanLa committed:
Merge branch 'release-msi-user-assigned' of https://github.com/microsoftdocs/azure-docs-pr into bryanla-release-msi-configure

BryanLa committed:
intial updates

johnkemnetz committed:
new eh stuff

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into eventhub-articles

BryanLa committed:
updates

tracsman committed:
Add TOC update, fix style formatting and one typo

johnkemnetz committed:
add single eh

Ja-Dunn committed:
Merge pull request #30243 from v-nagta/pluralsight

USER STORY 1147269 SaaS App Tutorial: Pluralsight Tutorial Update

tynevi committed:
Update .gitignore

Lisaco88 committed:
Merge pull request #29991 from JimacoMS2/dps-quick-service-tpm-node

Node service TPM enrollment quickstart

Commit:
blurred Subscription ID in portal URL

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-quick-service-x509-node
sync with master

Commit:
synced with master, blurred subscription ID in URL, and added TOC entry

johnkemnetz committed:
add single eh and multiple settings

Ja-Dunn committed:
Merge pull request #30555 from johnkemnetz/patch-2

add single eh

tynevi committed:
Merge pull request #30556 from MicrosoftDocs/tysonn-patch-4

Update .gitignore

Lisaco88 committed:
Merge pull request #30552 from BryanLa/bryanla-release-msi-configure

Intial updates for use / SDK and use / template

Ja-Dunn committed:
Merge pull request #30417 from johnkemnetz/eventhub-articles

Eventhub articles DRAFT

Ja-Dunn committed:
Merge pull request #30503 from tracsman/jonorTrouble

New doc on ExpressRoute troubleshooting

Ja-Dunn committed:
Merge pull request #30502 from hning86/master

register ACS RP

Ja-Dunn committed:
Merge pull request #30557 from johnkemnetz/patch-3

add single eh and multiple settings

v-jasowells committed:
Remove en-us from links.

v-jasowells committed:
Remove en-us from links.

v-jasowells committed:
Remove en-us from links.

v-jasowells committed:
Remove en-us from links.

v-jasowells committed:
Remove en-us from links.

v-alje committed:
Merge pull request #30280 from MicrosoftDocs/release-msi-user-assigned

Release msi user assigned

v-jasowells committed:
Remove en-us from links.

v-jasowells committed:
Fix casing.

v-jasowells committed:
Add line break before ```

v-jasowells committed:
Update tutorial-bing-entities-search-single-page-app-source.md

v-jasowells committed:
Update tutorial-bing-entities-search-single-page-app.md

v-alje committed:
Bringing even with master.

Ja-Dunn committed:
Merge pull request #30533 from BryanLa/bryanla-msi-add-ppinfo-to-banner

add pp info to preview banner

Ja-Dunn committed:
Merge pull request #30005 from JimacoMS2/dps-quick-service-x509-node

Node service X509 enrollment quickstart

v-jasowells committed:
Fixed next steps section.

BryanLa committed:
last minute ua cleanup

v-jasowells committed:
Update rank-results.md

Commit:
conclusion modification

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dps-tpm-device-java-selector

Ja-Dunn committed:
Merge pull request #30561 from BryanLa/bryanla-msi-uapp

Last minute ua cleanup

Commit:
RP instructions added

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

GitHubber17 committed:
Merge pull request #30562 from msebolt/dps-tpm-device-java-selector

Dps tpm device java selector

GitHubber17 committed:
Merge pull request #30568 from hning86/master

RP instructions added

GitHubber17 committed:
Merge pull request #30520 from linda33wj/master

Update Dynamics data type support

Commit:
Multi-VM-document

Commit:
mult-vm-2

Commit:
acrolinx score

Commit:
numbering

Commit:
troubleshoot-article

Commit:
alignment

Commit:
dinal-changes

wolfma61 committed:
Update GetStartedJSWebsockets.md

addressed comments by doc review/rebecca

Lisaco88 committed:
Merge pull request #30395 from puicchan/master

Updated to clarify deploy method is Git or FTP not both

Lisaco88 committed:
Merge pull request #30521 from wolfma61/patch-1

Update GetStartedJSWebsockets.md

Lisaco88 committed:
Merge pull request #30572 from sujayt/master

Multi-VM-document

v-alje committed:
Merge pull request #30473 from MicrosoftDocs/release-automation-mvc

Release automation mvc

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Update rank-results.md

v-alje committed:
Merge pull request #30575 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #29164 from v-jaswel/master

Add quickstarts.
